config_panel.toml

```diff
@@ -0,0 +1,25 @@
+version = "1.0"
+
+[main]
+name = "Castopod configuration"
+
+    [main.php_fpm_config]
+    name = "PHP-FPM configuration"
+
+        [main.php_fpm_config.fpm_footprint]
+        ask = "Memory footprint of the service?"
+        choices = ["low", "medium", "high", "specific"]
+        default = "low"
+        help = "low <= 20Mb per pool. medium between 20Mb and 40Mb per pool. high > 40Mb per pool.<br>Use specific to set a value with the following option."
+
+        [main.php_fpm_config.free_footprint]
+        ask = "Memory footprint of the service?"
+        type = "number"
+        default = "0"
+        help = "Free field to specify exactly the footprint in Mb if you don't want to use one of the three previous values."
+
+        [main.php_fpm_config.fpm_usage]
+        ask = "Expected usage of the service?"
+        choices = ["low", "medium", "high"]
+        default = "low"
+        help = "low: Personal usage, behind the sso. No RAM footprint when not used, but the impact on the processor can be high if many users are using the service.<br>medium: Low usage, few people or/and publicly accessible. Low RAM footprint, medium processor footprint when used.<br>high: High usage, frequently visited website. High RAM footprint, but lower on processor usage and quickly responding."
```

Comment:
Hmpf this is great and everything and i'm gonna be "that guy always overthinking everything", but this looks pretty close to what we predicted to be happening, a.k.a people adding this same config panel in every php apps ...

If the intent really is to add this to every PHP apps, we should really really **really** consider adding a mecanism in the core for a standard php-fpm config panel section instead of copy pasting this snippet in every app. Otherwise I already know how it's going to end up ... sooner or later we're going to want to change this, then there's gonna be 30+ apps implementing this, we're gonna have to update all of them, yet at the same time keep supporting the legacy for like 1 or 2 years, etc ...

Reply:
If only I could add a mechanism in the core that automatically adds this... doing it manually is the closest I can achieve 😅